features/promotion/receiving_fixed_discount_on_items_on_price_range.feature

```diff
@@ -0,0 +1,81 @@
+@promotion
+Feature: Receiving fixed discount on products from specific price range
+    In order to pay less while buying goods from specific price range
+    As a Customer
+    I want to receive discount for my purchase
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And the store has a product "PHP Mug" priced at "€20.00"
+        And there is a promotion "Christmas promotion"
+
+    @todo
+    Scenario: Receiving fixed discount on a item fulfilling minimum price criteria
+        Given this promotion gives "€10.00" off on every product more expensive than "€50.00"
```

Comment:
Given **the** promotion gives... `this` should be used when the preceding line `talk` about promotion.